### PR TITLE
Add primex online monitoring

### PR DESCRIPTION
**Pull request by igjaegle**
PrimEx-eta online monitoring:
1/ Plot for basic selection criteria 3 eta decays invariant mass: gg, pi0pi0pi0, and pi9pippim
2/ Elasticity & polar angles
3/ Some millecious plots (TOF veto, time, FCAL energy sum)

### CODE REVIEW

**Review by andrsmit — APPROVED** [review]

**Comment by gluex** [comment]
Test status for this pull request: SUCCESS
 
  Summary: [/work/halld/pull_request_test/halld_recon^ijaegle-primex-eta/tests/summary.txt](https://halldweb.jlab.org/pull_request_test/halld_recon^ijaegle-primex-eta/tests-summary.txt)
 Logs: [/work/halld/pull_request_test/halld_recon^ijaegle-primex-eta/tests/log](https://halldweb.jlab.org/pull_request_test/halld_recon^ijaegle-primex-eta/tests-logs)
 
 Build log: [/work/halld/pull_request_test/halld_recon^ijaegle-primex-eta/make_ijaegle-primex-eta.log](https://halldweb.jlab.org/pull_request_test/halld_recon^ijaegle-primex-eta/make_ijaegle-primex-eta.log)
 Build report: [/work/halld/pull_request_test/halld_recon^ijaegle-primex-eta/report_ijaegle-primex-eta.txt](https://halldweb.jlab.org/pull_request_test/halld_recon^ijaegle-primex-eta/report_ijaegle-primex-eta.txt)
 Location of build: /work/halld/pull_request_test/halld_recon^ijaegle-primex-eta

**Comment by aaust** [comment]
@somovalex You should wait for the successful test until you merge a pull request into the master.

@igjaegle In the current state, these macros cannot be included in rootspy. You can look at other examples on how to define the hnamepath for each histogram. I don't think, it is possible to use any include statements in a rootspy macro.

**Pull request merged**
